vyper/parser/parser.py

```diff
@@ -43,7 +43,11 @@
     raise Exception("Requires python 3.6 or higher for annotation support")
 
 # Header code
-STORE_CALLDATA: List[Any] = ['seq', ['mstore', 28, ['calldataload', 0]]]
+STORE_CALLDATA: List[Any] = \
+        ['seq',
+            ['if', ['lt', 'calldatasize', 4],
```

Comment:
Would be nice to add a note about why this is here.  To someone reading the code who's not familiar with the discussion, it might look mysterious.